server/package.json

```diff
@@ -40,6 +42,7 @@
   "scripts": {
     "dev": "nodemon main.js",
     "start": "node main.js",
+    "serve-openapi": "node main.js --openapi",
```

Comment:
@lilitkarapetyan - is the `serve-openapi` command there _just-in-case_?

So, the API docs are always there, but this command is to just serve the API docs.  Is that right?  If it is, we should add a some comment to the read-me.